src/components/mark/mark.styles.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import { UseEuiTheme, transparentize } from '../../services';
+
+export const euiMarkStyles = ({ euiTheme, colorMode }: UseEuiTheme) => {
+  // TODO: Global variables?
+  const transparency = { LIGHT: 0.1, DARK: 0.3 };
```

Comment:
Just to check, do we want to address this TODO now or in a later PR?

Reply:
We'll do it here. I've been meaning to ask @cchaos if these should be part of theme.

Reply:
Leave it hard-coded for now but update the comment to remind us what the variable was:
```suggestion
  // TODO: Was $euiFocusBackgroundColor
  const transparency = { LIGHT: 0.1, DARK: 0.3 };
```